pages/common/vim.md

```diff
@@ -23,9 +23,9 @@
 
 `<Esc>/{{search_pattern}}<Enter>`
 
-- Perform a regex substitution in the whole file (from the start, `1`, to the end, `$`):
+- Perform a regex substitution in the whole file (from the start to the end(% holds the file name)):
```

Comment:
We can remove the (from start to the end) part. That was needed to signify 1 -> $.

Reply:
I like this change, but I would remove everything under parenthesis. In this case it is not true that "% holds the file name". Actually it is a [range](http://vimdoc.sourceforge.net/htmldoc/cmdline.html#:range), and the most commonly used one.

Reply:
I thought holds the file name. When I type `:!echo %` it's also printing the file name.
But I've tried right now that I can't do smth like `:<file>s/pattern/pattern/modifier` - which makes sense. 
Fixed.

Reply:
@mstruebing, the vim documentation is very good. With [`:help :s`](http://vimdoc.sourceforge.net/htmldoc/change.html#:s) you see that it's actually:

    :[range]s[ubstitute]/{pattern}/{string}/[flags] [count]

Reply:
@mstruebing - Can you push your commit ? I dont see any updates.